### PR TITLE
Update HE endpoint

### PR DESCRIPTION
**Pull request by SerhiiBovtriuk**

### CODE REVIEW

**Review by oyvang — APPROVED** [review]

**Review by paulrene — APPROVED** [review]

File: connect/src/com/telenor/connect/utils/ConnectUrlHelper.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
     public static final String ACTION_ARGUMENT = "com.telenor.connect.ACTION_ARGUMENT";
     public static final String OAUTH_PATH = "oauth";
-    private static final String HE_TOKEN_API_BASE_PATH = "id/extapi/v1/header-enrichment-token/";
+    private static final String HE_TOKEN_API_BASE_PATH = "v2/extapi/v1/header-enrichment-token/";
```
Comment (paulrene) [inline connect/src/com/telenor/connect/utils/ConnectUrlHelper.java:27]: This new endpoint will not be available on the old connect hosts.

Comment (SerhiiBovtriuk) [reply connect/src/com/telenor/connect/utils/ConnectUrlHelper.java:27]: @paulrene it was confirmed that we are good to go with this change.

Comment (vlarsen) [reply connect/src/com/telenor/connect/utils/ConnectUrlHelper.java:27]: so, is it v2 or v1?

Comment (SerhiiBovtriuk) [reply connect/src/com/telenor/connect/utils/ConnectUrlHelper.java:27]: @vlarsen this is a trap for potentials hackers.

**Pull request merged**